requirements.txt

```diff
@@ -2,3 +2,4 @@ Django>=3.2.4,<3.3
 djangorestframework>=3.12.4,<3.13
 psycopg2>=2.8.6,<2.9
 drf-spectacular>=0.15.1,<0.16
+uwsgi>=2.0.21,<2.1
```

Comment:
Curious - was this needed over just using runserver?

Reply:
I added uwsgi while doing the security exercise. I figured it was safer to have an application server reject oversized payloads before DRF had a chance to parse it and cause an OOM.